corehq/apps/reminders/event_handlers.py

```diff
@@ -311,8 +309,7 @@ def fire_sms_survey_event(reminder, handler, recipients, verified_numbers):
             key = "start-sms-survey-for-contact-%s" % recipient.get_id
             with CriticalSection([key], timeout=60):
                 # Close all currently open sessions
-                XFormsSession.close_all_open_sms_sessions(reminder.domain,
-                    recipient.get_id)
+                SQLXFormsSession.close_all_open_sms_sessions(reminder.domain, recipient.get_id)
```

Comment:
Is the intent of this pr to only use the sql xforms sessions going forward? Because this line will make the sql sessions not agree with the couch ones anymore.